_posts/2017-04-04-optimising-performance-of-your-enterprise-angular-application.md

```diff
@@ -0,0 +1,267 @@
+---
+layout: post
+authors: [jan_de_wilde,frederic_ghijselinck,orjan_de_smet]
+title: "Optimising performance of your Enterprise Angular Application"
+image: /img/optimising-performance-of-your-enterprise-angular-application.png
+tags: [Angular]
+category: Angular
+comments: true
+---
+
+This blog post contains best practices that helped us optimise performance of our Enterprise Angular (v2+) Application we created for one of our clients.
+The project has been created in under 6 months with a dedicated team of 7 people of which 4 people are from JWorks (2 frontend, 2 backend) and consists of two Angular Applications that use modules and components from a shared library.
+The use of Angular Universal does not apply (yet) for this project.
```

Comment:
The Angular Universal part? We don't use Angular Universal for server side rendering of components. That's also a way of improving performance.